source/common/upstream/cluster_manager_impl.cc

```diff
@@ -479,6 +479,37 @@ bool ClusterManagerImpl::addOrUpdateCluster(const envoy::api::v2::Cluster& clust
       // Otherwise, applyUpdates() will fire with a dangling cluster reference.
       updates_map_.erase(cluster_name);
 
+      // If the active cluster has priority sets and the warming cluster does not have them, it
```

Comment:
There's only one priority set per cluster, so this wording is a bit strange

Reply:
Yeah, I meant hosts - will change